.eslintrc

```diff
@@ -20,6 +20,8 @@
     "sum": true,
     "count": true,
     "gapi": true,
+    "Set": true,
+    "ga": true,
```

Comment:
cool. can clean those up then. we good to merge? thoughts on the gold wedge size @bhollis ?